superset-frontend/src/explore/components/ExploreViewContainer.jsx

```diff
@@ -365,10 +365,20 @@ function ExploreViewContainer(props) {
     );
   }
 
+  function onResize(evt) {
+    const { x } = evt;
+    if (x > 0) {
+      localStorage.setItem('explore_sidebar_widths', x);
```

Comment:
You might want to check that localStorage is available or catch eventual errors. It might be disabled and will throw an exception in that case.